File: app_pojav_zh/src/main/java/com/movtery/ui/subassembly/customprofilepath/ProfilePathHome.java
package com.movtery.ui.subassembly.customprofilepath;

public class ProfilePathHome {
    public static String getGameHome() {
        return ProfilePathManager.getCurrentPath() + "/.minecraft";
    }

    public static String getVersionsHome() {
        return getGameHome() + "/versions";
    }

    public static String getLibrariesHome() {
        return getGameHome() + "/libraries";
    }

    public static String getAssetsHome() {
        return getGameHome() + "/assets";
    }

    public static String getResourcesHome() {
        return getGameHome() + "/resources";
    }
}

File: app_pojav_zh/src/main/java/com/movtery/pojavzh/feature/customprofilepath/ProfilePathHome.kt
package com.movtery.pojavzh.feature.customprofilepath

class ProfilePathHome {
    companion object {
        @JvmStatic
        val gameHome: String
            get() = ProfilePathManager.currentPath + "/.minecraft"

        @JvmStatic
        val versionsHome: String
            get() = "$gameHome/versions"

        @JvmStatic
        val librariesHome: String
            get() = "$gameHome/libraries"

        @JvmStatic
        val assetsHome: String
            get() = "$gameHome/assets"

        @JvmStatic
        val resourcesHome: String
            get() = "$gameHome/resources"
    }
}
